content/about/faq.md

```diff
@@ -2,48 +2,86 @@
 title : "Frequently Asked Questions"
 page_header_bg : "images/background/page-title-bg.jpg"
 date: 2024-11-06T20:41:24+02:00
-description : "The Free and Open Source Software for Geospatial Europe (FOSS4GE) conference is the European branch event of the Open Source Geospatial Foundation (OSGeo) annual conference."
+description : "The Free and Open Source Software for Geospatial Europe (FOSS4GE)
+conference is the European branch event of the Open Source Geospatial Foundation
+(OSGeo) annual conference."
 draft : false
 layout : "faq"
 ---
 
 
 {{< faq "When and where will the conference take place?" >}}
-The conference will take place from 14 - 20 July 2025 in Mostar, Bosnia-Herzegovina.  
+The conference will take place from 14 - 20 July 2025 in Mostar,
+Bosnia-Herzegovina.  
 {{</ faq >}}
 
 {{< faq "If the conference is in Bosnia-Herzegovina, does it mean my presentation/workshop needs to be in Bosnian?" >}}
-No. The working language of the conference is English.
+No, the official language of the conference is English. All presentations and
+workshops should be conducted in English to accommodate our international
+audience.
 {{</ faq >}}
 
 {{< faq "Will the presentations be public after the conference?" >}}
-Yes. Our main goal is to foster FOSS4G, and that means also making all the presentations available even to people that didn’t register in the first place. After the conference, the recorded talks will be published on our YouTube channel - https://www.youtube.com/@FOSS4G  
+Yes, all presentations will be recorded and made publicly available after the
+conference. Our main goal is to foster FOSS4G, and that means also making all
+the presentations available even to people that didn’t register in the first
+place. After the conference, the recorded talks will be published on our
+YouTube channel - https://www.youtube.com/@FOSS4G  
 {{</ faq >}}
 
 {{< faq "Have you considered a hybrid online-face to face event?" >}}
-FOSS4GE 2025 will be an in-person event.  
+Yes, FOSS4G Europe 2025 is organized as an in-person event. However, to
```

Comment:
This reply does not make it clear what is the plan. Is the plan to only stream presentations and not allow for virtual presentations? Or are you aiming for hybrid too (meaning presentations also possible online)?

If it's only streaming then I would not say "Yes" as an answer to a question if the event is online